packages/legend-studio/src/stores/EditorStore.ts

```diff
@@ -796,6 +798,10 @@ export class EditorStore {
         ),
       ])) as [Entity[], PlainObject<ProjectConfiguration>];
       entities = result[0];
+      // NOTE: if graph entities are provided, we will use that to build the graph.
+      // We use this method as a way to fully reset the application with the entities, but we still use
+      // the workspace entities for hashing as those are the base entities.
+      entities = result[0];
```

Comment:
Please remove this duplicated line, also let's move this comment to where you call `buildGraph`, i.e. `L827`